dbconn/dbconn.go

```diff
@@ -194,8 +194,15 @@ func (dbconn *DBConn) Connect(numConns int) error {
 	if dbconn.ConnPool != nil {
 		return errors.Errorf("The database connection must be closed before reusing the connection")
 	}
-	// This string takes in the literal user/database names. They do not need to be escaped or quoted.
-	connStr := fmt.Sprintf("postgres://%s@%s:%d/%s?sslmode=disable", dbconn.User, dbconn.Host, dbconn.Port, dbconn.DBName)
+	// This string takes in the literal user/database names. They do not need
+	// to be escaped or quoted.
+	// By default pgx/v4 by default turns on automatic prepared statement
```

Comment:
"By default" twice here.  Remove the second one... unless this is a super default then it's okay. 😄